http_source.go

```diff
@@ -11,6 +11,8 @@ import (
 	"text/template"
 )
 
+// HTTPSource provide a Source that will download the update from a HTTP url.
+// It is expecting the signature expected to be at ${URL}.ed25519
```

Comment:
This sentence is a bit weird. Does it mean that there should be another file expected at that location containing an ED25519 signature?

Reply:
Yes, indeed.